test/src/test/java/org/corfudb/integration/StreamingIT.java

```diff
@@ -1345,7 +1174,7 @@ public void testSubscribeAPIForTagsOnly() throws Exception {
                 Timestamp.newBuilder().setSequence(Address.NON_ADDRESS).setEpoch(0L).build());
 
         // Confirm totalUpdates are received
-        TimeUnit.MILLISECONDS.sleep(sleepTime);
+        TimeUnit.MILLISECONDS.sleep(sleepTime * 2);
```

Comment:
In certain cases it is slower, because we are not continuously polling there could be some lag. But these tests are failing even without this change. Here's a failure from another PR https://pipelines.actions.githubusercontent.com/NnzYamcbKeYKYrtwq5VEoRtMbxsjokL1h7qBUldebVzgKpZkWn/_apis/pipelines/1/runs/2638/signedlogcontent/8?urlExpires=2021-10-04T20%3A42%3A16.4698112Z&urlSigningMethod=HMACV1&urlSignature=XjzBqYX9O1wjs7xPT4%2BMj0%2BLRrCslJx7ATtH7tC4uFY%3D